### DISCUSSION TITLE
Download button

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#4309] [Ideas]
How about a download project button for the outside for the project to download to cpu!

**Comment by CST1229** [comment]
What? Do you mean adding an SB3 save to your computer button to the project page?
Or adding a button to convert your project to .exe and download it?

**Reply by LilSefGaming** [reply]
I mean a SB3, like as when you save a project into a computer, but a quicker way, without clicking "see inside"

**Reply by WojtekCodesToday** [reply]
I see what are you trying... ( It's just adding a button, just like the turbowarp button)

**Comment by LilSefGaming** [comment]
So that means ur supporting?

**Comment by Dhuls-Scratch** [comment]
clarify.
 how do you download to a CPU?
If you want to download as a .exe or .zip, use https://packager.turbowarp.org

**Reply by CST1229** [reply]
> I mean a SB3, like as when you save a project into a computer, but a quicker way, without clicking "see inside"

And I think by CPU they mean the computer.
